packages/cli/src/commands/package-dynamic-plugins/command.ts

```diff
@@ -121,6 +135,27 @@ export async function command(opts: OptionValues): Promise<void> {
             keywords,
           },
         });
+        // some plugins include configuration snippets in an app-config.janus-idp.yaml
+        const pluginConfigPath = path.join(
+          packageDirectory,
+          'app-config.janus-idp.yaml',
```

Comment:
Is the convention to use the organization or repo name here?  I've seen some plugins in the community plugins repo that have this file named `app-config.community-plugins.yaml`  Maybe this will need to be more flexible.

Reply:
I went and added a function to the first commit that will look for a few possible known files, we can always refine this later.